js/views/pageNavVw.js

```diff
@@ -350,18 +350,19 @@ module.exports = Backbone.View.extend({
     $('#obContainer').addClass('blur');
 
     // display the modal
-    $('.js-aboutModal').removeClass('hide');
+    $('.js-aboutModal').removeClass('fadeOut').addClass('fadeIn');
   },
```

Comment:
It would be nice to get them all working the same way if it's relatively simple.